data/bactrian.txt

```diff
@@ -0,0 +1,419 @@
+	landing
+	source Farpoint
+	to offer
+		has "Bactrian Unlock [0]: active"
+		not "found star queen"
+	on offer
+		conversation
+			`After speaking with a Navy Officer for a few minutes, you find out that the Star Queen came through here over a week ago, escorted by several large pirate ships. The Navy did not want to risk the lives of those on the Star Queen by engaging the pirate fleet, so they let it continue north from here.`
+			`	The spaceport clearly shows preparations for a rescue mission, but due to the destruction of a number of Navy capital ships by the Star Queen's captors in the days preceding its abduction, they have been ordered to wait for additional ammunition and reinforcements.`
+			`	The Navy officer ends by hinting that the Star Queen is likely to be in Alnilam or Arneb, as those systems are the farthest from anyone looking to find or rescue the ship.`
+				decline
+
+mission "Bactrian Unlock [0]: Pirates"
+	invisible
+	landing
+	destination Valhalla
+	to offer
+		has "Bactrian Unlock [0]: active"
+	to fail
+		or
+			has "Bactrian Unlock [3]: done"
+			has "Bactrian Unlock [3-alt]: offered"
+	npc kill
+		personality nemesis staying
+		system Arneb
+		government Pirate
+		ship "Leviathan (Laser)" "His Right Hand"
+		ship "Leviathan (Laser)" "His Left Hand"
+
+mission "Bactrian Unlock [0]: Haven"
+	landing
+	source Haven
+	to offer
+		has "Bactrian Unlock [0]: active"
+	on offer
+		conversation
+			`You spend a few hours scanning the surface of the planet in an attempt to find the Star Queen, but nothing comes up. It will likely require military-grade scanners in order to find the ship, which the Deep Security will undoubtedly have.`
+				decline
+
+mission "Bactrian Unlock [1]"
+	name "Secure the Scientists"
+	description "Escort the <npc> to <destination> to retrieve the Star Queen. Escorts will be waiting for you on Memory in the Zosma system and Farpoint in the Alnitak system."
+	landing
+	destination Haven
+	clearance
+	to offer
+		has "Bactrian Unlock [0]: done"
+	on offer
+		conversation
+			`You return to Valhalla and find the man that you met earlier in the spaceport talking with a large group of people in Deep Security uniforms.`
+			`	As you approach them, the man stops talking with the group after noticing you. "Captain <last>! Have you located the Star Queen?"`
+			choice
+				`	"Yes, it is located on <destination>, but is heavily defended."`
+			
+			branch destroyed
+				has "Bactrian Unlock [0]: Pirates: done"
+			`	You mention the multiple heavy warships that you saw in the system, and the fact that there are probably many more waiting on Haven.`
+				goto next
+			
+			label destroyed
+			`	You mention the multiple heavy warships that you took out in the system, but that there may be many more waiting on Haven.`
+			
+			label next
+			`	After hearing this news, many of the faces in the group turn grim as they realize that they will need to fight to retrieve the Star Queen. `
```

Comment:
> After speaking with a Navy Officer for a few minutes

Line 115: Shouldn't _officer_ be lowercase? In other places it is.

> Unfortunately, there are also a large number of pirate vessels headed in your direction.  You should report back to the Deep Security officer on <destination> for further instruction.

Line 84: Double space after period (should be single space).